.github/workflows/format-rules.yaml

```diff
@@ -0,0 +1,30 @@
+name: format-rules
+
+on:
+  push:
+    branches:
+      - develop
+      - release
+
+jobs:
+  test-latest:
+    name: format-rules
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "17.3.0"
+      - name: install prettier
+        run: npm install prettier
+      - name: format unformatted rules
+        run: npx prettier --write $(npx prettier --list-different "./**/*.yaml")
+      - name: commit formatted files
+        run: |
+          date > generated.txt
```

Comment:
What's the thinking behind this file?

Reply:
Oh that's a good point. That's not necessary. I'll remove that. I also noticed that the action fails when no files are changed, so I'll add a check for that too in my next push.

Reply:
👍  Thanks!